README.md

```diff
@@ -1,5 +1,230 @@
-# iota-sdk
+# IOTA SDK Library
 
 [![Coverage Status](https://coveralls.io/repos/github/iotaledger/iota-sdk/badge.svg?branch=develop)](https://coveralls.io/github/iotaledger/iota-sdk?branch=develop)
 
-WIP
+The IOTA SDK is a Rust-based project that provides a convenient and efficient way to interact with Shimmer nodes in the
+Shimmer network. It consists of two main modules: `wallet` and `client`.
```

Comment:
> Shimmer nodes in the Shimmer network
At some point it will be also IOTA nodes and it's also not limited to the Shimmer network, testnet works too
So maybe just mention stardust protocol with https://wiki.iota.org/shimmer/develop/explanations/what-is-stardust ?